cookiecutter/vcs.py

```diff
@@ -7,7 +7,10 @@
 import os
 import subprocess
 
-from whichcraft import which
+try:
+    from shutil import which
+except ImportError:
+    from whichcraft import which
```

Comment:
@anntzer I'm a little confused by this, isn't the point of `whichcraft` that you _don't_ have to do this? 🤔

Reply:
I don't think saving three lines of code is worth an additional dependency, but ultimately it's not my call.